### PR TITLE
Dev 45139 grafana x validations UI updates

### PR DESCRIPTION
**Pull request by copyhold**
hide grafana logo
hide help menu
hide datasources menu
hide language select from profile

## before 

image

## After

image

### CODE REVIEW

File: custom.ini [hunk 1]
```diff
@@ -82,3 +82,6 @@ min_refresh_interval = 30s
 [log.frontend]
 enabled = true
 custom_endpoint = log
+
+[feature_toggles]
+dashgpt = false
```
Comment (ifatregev) [inline custom.ini:87]: what is this?

File: pkg/services/navtree/navtreeimpl/navtree.go [hunk 2]
```diff
@@ -485,7 +485,7 @@ func (s *ServiceImpl) buildDataConnectionsNavLink(c *contextmodel.ReqContext) *n
 
 	baseUrl := s.cfg.AppSubURL + "/connections"
 
-	if hasAccess(datasources.ConfigurationPageAccess) {
+	if false && hasAccess(datasources.ConfigurationPageAccess) { // LOGZ.IO GRAFANA CHANGE :: hide datasources menu, should be done by altering permissions
```
Comment (ifatregev) [inline pkg/services/navtree/navtreeimpl/navtree.go:488]: if false? haha why?

Comment (copyhold) [reply pkg/services/navtree/navtreeimpl/navtree.go:488]: hide the datasources menu

File: custom.ini (hunk 1 above)
Comment (copyhold) [reply custom.ini:87]: makes the gpt grafana plugins not load

**Review by ifatregev — APPROVED** [review]
ok

**Pull request merged**